src/data/AmendmentCenter.hpp

```diff
@@ -127,6 +127,10 @@ struct Amendments {
     REGISTER(fixInnerObjTemplate2);
     REGISTER(fixNFTokenPageLinks);
     REGISTER(InvariantsV1_1);
+    REGISTER(Credentials);
+    REGISTER(AMMClawback);
+    REGISTER(fixAMMv1_2);
+    REGISTER(MPTokensV1);
```

Comment:
nit: we should reserve the order

Reply:
Not important, but the following order is how rippled orders them and the sequence they were merged:
```
    REGISTER(MPTokensV1);
    REGISTER(fixAMMv1_2);
    REGISTER(AMMClawback);
    REGISTER(Credentials);
```

(also i believe this is the way it's ordered in the MPT and credential clio PRs)